File: src/main/java/pl/arturczopek/mycoach/repository/ExerciseRepository.java
package pl.arturczopek.mycoach.repository;

import org.springframework.data.repository.CrudRepository;
import pl.arturczopek.mycoach.model.database.Exercise;

/**
 * @Author Artur Czopek
 * @Date 09-10-2016
 */

public interface ExerciseRepository extends CrudRepository<Exercise, Long> {

    void deleteBySetId(long setId);
}

File: src/main/kotlin/pl/arturczopek/mycoach/repository/ExerciseRepository.kt
package pl.arturczopek.mycoach.repository

import org.springframework.data.repository.CrudRepository
import pl.arturczopek.mycoach.model.database.Exercise

/**
 * @Author Artur Czopek
 * @Date 09-10-2016
 */

interface ExerciseRepository : CrudRepository<Exercise, Long>
